steps/cadence-innovus-power-includes-sky130/outputs/main.tcl

```diff
@@ -0,0 +1,37 @@
+#=========================================================================
+# main.tcl
+#=========================================================================
+# A generic power strategy that should work for most designs
+#
+# Author : Christopher Torng
+# Date   : January 13, 2020
+
+#-------------------------------------------------------------------------
+# Implement power strategy
+#-------------------------------------------------------------------------
+# Older technologies use a single coarse power mesh, but more advanced
+# technologies often use a combination of a fine+coarse power mesh.
+#
+# Here we check the direction of M2 to decide which power strategy to use.
+
+set M2_direction [dbGet [dbGet head.layers.name 3 -p].direction]
```

Comment:
This used to be 2 and not 3:

```
< set M2_direction [dbGet [dbGet head.layers.name 2 -p].direction]
---
> set M2_direction [dbGet [dbGet head.layers.name 3 -p].direction]
```

If we can fix everything else, I would like this to be a little SKY130 switch... or a parameter.

Reply:
Better than having a separate custom power package for sky130